docs/pages/upgrades/v4.mdx

```diff
@@ -0,0 +1,106 @@
+# Upgrade
+
+Version 4: Welcome to Typescript!
+
+## Typescript!
+
+We are so excited to add Typescript on Welcome UI 😍
+
+## Breaking Changes
+
+### xStyled
+
+We upgraded xStyled core libraries to V3.
+
+- Use `w="100%"` instead of `w={1}`
+- Use `h="100%"` instead of `h={1}`
+
+Check the migration here: [Changelog of xstyled v3](https://xstyled.dev/docs/upgrade-guide/)
+
+### Theme
+
+#### Colors
+
+- `theme.colors.success[500]` becomes `theme.colors.success[700]`
+- `theme.colors.success[200]` becomes `theme.colors.success[500]`
+- `theme.colors.info[500]` becomes `theme.colors.info[700]`
+- `theme.colors.info[200]` becomes `theme.colors.info[500]`
+- for welcome theme:
+  - `theme.colors.success[200]` has been changed for the new color `#45BE98`
+  - `theme.colors.info[200]` has been changed for the new color `#9AC7F7`
+- for the core theme:
+  - `theme.colors.success[200]` has been changed for the new color `#7EC784`
+  - `theme.colors.info[200]` has been changed for the new color `#6696DF`
+
+#### textsTextTransform
+
+- removed uppercase on `h0`
+- add uppercase on `subtitle1` and `subtitle2`
+
+### Icons
+
+- 1 icon deleted (doublon with `EditIcon`)
+  - `pen`
+- 12 new icons added:
+  - `avatar_accessory`
+  - `avatar_body`
+  - `avatar_bottom`
+  - `avatar_costume`
+  - `avatar_head`
+  - `avatar_top`
+  - `common_area`
+  - `meeting_room`
+  - `bookmark`
+  - `bookmark_outline`
+  - `equal`
+- 2 icons modified :
+  - `screenshare_off`
+  - `screenshare_on`
+
+### Text component
+
+We reduce the size (and variant) of headings on mobile :
+
+- h0 > h1
+- h1 > h2
+- h2 > h3
+- h3 > h4
+- h4 > h5
+- h5 > h6
```

Comment:
Not sure what this means ? h0 is replaced with h1 size, and so on?

Reply:
Don't have precision about this modification (not related to this PR). It was a previous changes due to spacing modification.